frontend/src/components/common/notification/notification.tsx

```diff
@@ -16,6 +16,8 @@ const Notification: FC<ModalProps> = (props) => {
 
   const closeNotification = (): void => setIsOpen(!isOpen);
 
+  isOpen && setTimeout(closeNotification, 6000);
```

Comment:
All the side effects must be declared and cleared inside the `useEffect`. Because now on each rerender a new timeout will be created